src/chat/chatter.py

```diff
@@ -183,12 +183,27 @@ def update(self):
         # Status icon handling
         url = client.instance.urls.get(player.login)
         if url:
-            if url.scheme() == "fafgame":
-                self.statusItem.setIcon(util.icon("chat/status/lobby.png"))
-                self.statusItem.setToolTip("In Game Lobby<br/>"+url.toString())
-            elif url.scheme() == "faflive":
-                self.statusItem.setIcon(util.icon("chat/status/playing.png"))
-                self.statusItem.setToolTip("Playing Game<br/>"+url.toString())
+            if int(url.queryItemValue("uid")) in client.instance.games.games:
+                game = client.instance.games.games[int(url.queryItemValue("uid"))]
+                game_str = "  " + game.mod + "  on  '" + game.mapdisplayname + "'  (" + str(game.uid) + ") "
+                if url.scheme() == "fafgame":
+                    game_str = " Game Lobby:  '" + game.title + "'" + game_str
+                    if game.host == self.name:
+                        self.statusItem.setIcon(util.icon("chat/status/host.png"))
+                        self.statusItem.setToolTip("Hosting" + game_str)
+                    else:
+                        self.statusItem.setIcon(util.icon("chat/status/lobby.png"))
+                        self.statusItem.setToolTip("In" + game_str)
+                elif url.scheme() == "faflive":
+                    if time.time() - game.launched_at > 5 * 60:
+                        self.statusItem.setIcon(util.icon("chat/status/playing.png"))
+                        self.statusItem.setToolTip("Playing" + game_str)
+                    else:
+                        self.statusItem.setIcon(util.icon("chat/status/playing_delay.png"))
+                        self.statusItem.setToolTip("Playing" + game_str + " -  LIVE DELAY (5 Min)")
+            else:  # that shouldn't happen - but it does in rare cases
+                self.statusItem.setIcon(util.icon("chat/status/status_unclear.png"))
+                self.statusItem.setToolTip("(has url - but no matching running game found)<br/>" + url.toString())
```

Comment:
This became a lot of cases IMO (6 up from 3), we could try decoupling figuring out which state to choose from setting stuff up. Maybe something along the lines of:
```
class ChatterGameState(Enum):
    IDLE = ...
    HOST = ...
    LOBBY = ...
```
With some function like:
```
def setChatterGameState(self, state, url, game = None)
```
that does the setting depending on the state.
It's mostly nitpicking that could probably get fixed after other refactors, but would be nice to have.